### DISCUSSION TITLE
Doris运行过程中突然挂掉

### DISCUSSION THREAD

**Discussion by Tangsonghuai** [apache/doris#20160] [RFC]
现有日志查不到原因
<img width="1400" alt="截屏2023-05-29 下午3 16 47" src="https://github.com/apache/doris/assets/34538205/7241cf2c-7dfa-41f3-8504-4e1d6c075cbc">
<img width="1438" alt="截屏2023-05-29 下午3 17 07" src="https://github.com/apache/doris/assets/34538205/4bdbc6a4-3791-4f56-87ce-6dee136f2d9a">

**Comment by LemonLiTree** [comment]


**Reply by Tangsonghuai** [reply]
ok，Get!
